markdown/bitburner.ns.hackanalyzethreads.md

```diff
@@ -29,7 +29,7 @@ The number of threads needed to hack the server for hackAmount money.
 
 RAM cost: 1 GB
 
-This function returns the decimal number of script threads you need when running the hack command to steal the specified amount of money from the target server. If hackAmount is less than zero or greater than the amount of money available on the server, then this function returns -1.
+This function returns the decimal number of script threads you need when running the hack command to steal the specified amount of money from the target server. If hackAmount is less than zero, greater than the amount of money available on the server, or your hacking level is below the required level for the target server, then this function returns -1.
```

Comment:
This one caught me by surprise, before I realized that this is the first PR you made after I merged that other one. :D